server_installation/topics/operating-mode/crossdc/proc-configuring-remote-cache.adoc

```diff
@@ -0,0 +1,114 @@
+
+[id="proc-configuring-remote-cache-{context}"]
+= Configuring Remote Cache Stores on {project_name}
+After you set up remote {jdgserver_name} clusters to back up {jdgserver_name} data, you can configure the Infinispan subsystem  to use those clusters as remote stores.
+
+.Prerequisites
+
+* Set up remote {jdgserver_name} clusters that can back up {jdgserver_name} data.
+* Create a truststore that contains the SSL certificate with the {jdgserver_name} Server identity.
+
+.Procedure
+
+. Add the truststore to the {jdgserver_name} deployment.
```

Comment:
Add the truststore to the RHSSO deployment.

Reply:
I will fix that on the next commit.

Reply:
fixed. thanks, Andy